src/bincode/hashmap.zig

```diff
@@ -73,10 +73,11 @@ pub fn hashMapFieldConfig(
         }
 
         fn free(allocator: std.mem.Allocator, data: anytype) void {
+            var copy = data;
```

Comment:
why do you do this?

Reply:
It allows simplification of the bincode.free code. See the change in bincode.zig starting from 550.